packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -261,10 +293,14 @@ export class DataVisualizerGraph
 	): Promise<FluidObjectId | undefined> {
 		const resolvedObject = await handle.get();
 
+		if (isPureDataObject(resolvedObject)) {
```

Comment:
Changed from `isDataObject()` to `isPureDataObject()` since `registerVisualizerForVisualizableObject()` accepts `PureDataObject | ISharedObject`.